code/modules/mod/modules/modules_maint.dm

```diff
@@ -48,12 +48,60 @@
 		return
 	mod.wearer.visible_message("[src] inside [mod.wearer]'s [mod.name] snaps shut, mutilating the user inside!", span_userdanger("*SNAP*"))
 	mod.wearer.emote("scream")
+	mod.wearer.Paralyze(10000, TRUE)
 	playsound(mod.wearer, 'sound/effects/snap.ogg', 75, TRUE, frequency = 0.5)
 	playsound(mod.wearer, 'sound/effects/splat.ogg', 50, TRUE, frequency = 0.5)
 	mod.wearer.client?.give_award(/datum/award/achievement/misc/springlock, mod.wearer)
-	mod.wearer.apply_damage(500, BRUTE, forced = TRUE, spread_damage = TRUE, sharpness = SHARP_POINTY) //boggers, bogchamp, etc
+	mod.wearer.apply_damage(damage = 45, damagetype = BRUTE, def_zone = BODY_ZONE_R_ARM, forced = FALSE, spread_damage = FALSE, sharpness = WOUND_BLUNT)
+	ADD_TRAIT(mod, TRAIT_NODROP, HAND_REPLACEMENT_TRAIT)
+	addtimer(CALLBACK(src, .proc/stage_2), rand(8 SECONDS, 10 SECONDS))
```

Comment:
8 to 10 seconds 6 times can lead to up to a full minute of dying, which obviously is ridiculous. Am I alone in that?

Reply:
I mean, it's a slow and painful death.  You're already committing yourself to a very flashy and extravagant death, might as well make it last.

Reply:
very slow and painful death is the objective

Reply:
time between stage now shorten to 3 or 5